src/codemodder/codemodder.py

```diff
@@ -97,13 +97,19 @@ def analyze_files(
             line_include,
             sarif_for_file,
         )
-
-        apply_codemod_to_file(
-            execution_context,
-            file_context,
-            codemod,
-            source_tree,
-        )
+        try:
```

Comment:
I don't think there's anything wrong with this change (and I know we discussed it at some other point) but right now it makes me a little concerned that we might cover up any weird edge cases we've missed and so I'd prefer to let this continue to fail loudly for now.

Reply:
Sounds reasonable and I feel similarly, I just wanted to follow what codetf said which was "record failure if parsing or transformation fails". I can revert

Reply:
Yeah I see now where the spec says that. Still, let's hold off for now and revisit once exceptional behavior is a bit better defined. I think the parser error case makes sense since it's almost certainly out of our control (e.g. a malformed input file) but this case is more likely to be our own mistake.